src/components/DateSeparator/DateSeparator.js

```diff
@@ -7,9 +8,15 @@ import { withTranslationContext } from '../../context';
  * DateSeparator - A simple date separator
  *
  * @example ../../docs/DateSeparator.md
+ * @type {React.FC<import('types').DateSeparatorProps>}
  */
-const DateSeparator = ({ position, tDateTimeParser, formatDate, date }) => {
-  if (!Date.parse(date)) return null;
+const DateSeparator = ({
+  position = 'right',
```

Comment:
does the styleguidist pick these default values?

Reply:
Good call, not sure.

Reply:
If not, then lets not remove `DateSeparator.defaultProps` part

Reply:
![image](https://user-images.githubusercontent.com/1172870/85018071-dc5eb500-b16c-11ea-8d0a-9d360ab414d9.png)

picks them up

Reply:
Noice!!